src/languages/shell.js

```diff
@@ -17,7 +17,7 @@ export default function(hljs) {
         // We cannot add \s (spaces) in the regular expression otherwise it will be too broad and produce unexpected result.
         // For instance, in the following example, it would match "echo /path/to/home >" as a prompt:
         // echo /path/to/home > t.exe
-        begin: /^\s{0,3}[/~\w\d[\]()@-]*[>%$#]/,
+        begin: /^\s{0,3}[/~\w\d[\]()@-]*[>%$#]\s?/,
```

Comment:
Are there cases where this is too board?  I think this will also match tabs and newlines... I think perhaps `[ ]?` would be better?

Reply:
Yeah, good call. I changed it to that.